### PR TITLE
feat: add autodiscovery of cluster urls

### PR DESCRIPTION
**Pull request by jjaferson**
https://issues.jboss.org/browse/INTLY-3133

### CODE REVIEW

File: test/e2e/integreatly_test.go [hunk 1]
```diff
@@ -86,8 +83,24 @@ func integreatlyManagedTest(t *testing.T, f *framework.Framework, ctx *framework
...
+			Name:      "console",
+			Namespace: "openshift-console",
+		},
+	}
+	key := client.ObjectKey{
+		Name:      consoleRouteCR.GetName(),
+		Namespace: consoleRouteCR.GetNamespace(),
+	}
+
+	// consoleRoute, err := bootstrapReconciler.getConsoleRouteCR(goctx.TODO(), f.Client)
```
Comment (damienomurchu) [inline test/e2e/integreatly_test.go]: remove this

**Comment by damienomurchu** [comment]
Code looks good, will keep an eye on e2e tests but once they pass I think this is ready to be merged. I would just one remove the commented line before merging though.

File: test/e2e/integreatly_test.go (hunk 1 above)
Comment (jjaferson) [reply test/e2e/integreatly_test.go]: Yup, I totally missed that one

**Comment by damienomurchu** [comment]
@philbrookes just doing a final verification on this PR - any objection to letting this get automerged if the verification is ok and I remove the `hold`? Also, not sure what the release process for the integreatly-operator is now - is anything required for a new release?

**Pull request merged**